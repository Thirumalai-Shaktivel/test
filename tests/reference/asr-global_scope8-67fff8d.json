{
    "basename": "asr-global_scope8-67fff8d",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope8.f90",
    "infile_hash": "899df30dee274279a699110fb9af2afad0b826c0abc886f9e2460627",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope8-67fff8d.stdout",
    "stdout_hash": "ea3c50f51a43cddc9f2c8c14de368f986505761e717cbe6210c5cbe8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}