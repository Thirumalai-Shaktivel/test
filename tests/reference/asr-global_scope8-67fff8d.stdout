(TranslationUnit (SymbolTable 1 {x: (Variable 1 x Local () Default (Integer 4 []) Source Public Required)}) [(= (Var 1 x) (ConstantInteger 6 (Integer 4 []))) (Var 1 x)])
